firstvoices/backend/search/es_logging.py

```diff
@@ -14,18 +14,19 @@ def log_connection_error(e, instance):
 
 
 def log_connection_error_details(e, type_name, instance_id):
-    logger.error(ES_CONNECTION_ERROR, type_name, type_name, instance_id)
-    logger.error(e)
+    logger.error(
+        ES_CONNECTION_ERROR + "Error: [%s]", type_name, type_name, instance_id, e
+    )
```

Comment:
I appended the extra info here instead of in the constant because the constant is currently used in other places as well. (Same for the rest, below.)